src/main/java/core/basesyntax/dao/FruitDao.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Fruit;
+import java.util.List;
+
+public interface FruitDao {
+    void addCsvRow(Fruit fruitTransaction);
+
+    List<String> getCsv();
+
+    void createReportCsv(List<String> transactions);
+
+    public List<String> getReportCsv();
```

Comment:
Dao should have only methods that are related to the storage of internal data.
Basic setup of Dao is `create`, `read`, `update`, `delete` of records in our storage.
Please extract method that aimed to work with incoming file or report file to separate services like FileReader, FileWriter